src/Faker/Provider/hu_HU/Person.php

```diff
@@ -16,9 +16,8 @@ class Person extends \Faker\Provider\Person
         '{{title}} {{lastName}} {{firstNameFemale}}',
         '{{lastName}} {{firstNameFemale}} {{suffix}}',
         '{{title}} {{lastName}} {{firstNameFemale}} {{suffix}}',
-        // ..né
-        '{{lastName}} {{firstNameMaleNe}}',
-        '{{title}} {{lastName}} {{firstNameMaleNe}}',
+        '{{lastNameFemaleMarried}} {{$lastName}} {{firstNameFemale}}',
```

Comment:
Could you link to some documentation regarding hungarian name formats?

Reply:
Yes you can read about here: https://en.wikipedia.org/wiki/Hungarian_names

Reply:
Sorry for the misunderstanding, I meant inside a phpdoc `@see`.

See other providers for an example.

Reply:
Okay :)